site/docs/hive.md

```diff
@@ -17,117 +17,350 @@
+You can enable auto-conversion through Hadoop configuration (not enabled by default):
+
+| Config key                               | Default                     | Description                                         |
+| -----------------------------------------| --------------------------- | --------------------------------------------------- |
+| iceberg.mr.schema.auto.conversion        | false                       | if Hive should perform type auto-conversion         |
+
+### Hive type to Iceberg type
+
+This type conversion table describes how Hive types are converted to the Iceberg types.
+The conversion applies on both creating Iceberg table and writing to Iceberg table via Hive.
+
+| Hive             | Iceberg                 | Notes |
+|------------------|-------------------------|-------|
+| boolean          | boolean                 |       |
+| short            | integer                 | auto-conversion |
+| byte             | integer                 | auto-conversion |
+| integer          | integer                 |       |
+| long             | long                    |       |
+| float            | float                   |       |
+| double           | double                  |       |
+| date             | date                    |       |
+| timestamp        | timestamp without timezone |    |
+| timestamplocaltz | timestamp with timezone | Hive 3 only |
+| interval_year_month |                      | not supported |
+| interval_day_time |                        | not supported |
+| char             | string                  | auto-conversion |
+| varchar          | string                  | auto-conversion |
+| string           | string                  |       |
+| binary           | binary                  |       |
+| decimal          | decimal                 |       |
+| struct           | struct                  |       |
+| list             | list                    |       |
+| map              | map                     |       |
+| union            |                         | not supported |
+
+### Iceberg type to Hive type
+
+This type conversion table describes how Iceberg types are converted to the Hive types. 
+The conversion applies on reading from Iceberg table via Hive.
+
+| Iceberg                    | Hive                    | Note          |
+|----------------------------|-------------------------|---------------|
+| boolean                    | boolean                 |               |
+| integer                    | integer                 |               |
+| long                       | long                    |               |
+| float                      | float                   |               |
+| double                     | double                  |               |
+| date                       | date                    |               |
+| time                       | string                  |               |
```

Comment:
Not 100% sure about the type from iceberg to hive, I am writing this based on the object inspectors in the MR package. Please confirm if this conversion is correct, thanks! @marton-bod @pvary